### ISSUE TITLE
Qt::AA_EnableHighDpiScaling must be set before QCoreApplication is created

### ISSUE THREAD

**Issue by f4exb** [f4exb/sdrangel#522]
This is the message issued at the very start of the application at least from !t 5.12 and may explain why the DPI scaling does not actually works.

in `app/main.cpp` this is the present code at the very start:
```c++
	QApplication a(argc, argv);
/*
	QTextCodec::setCodecForTr(QTextCodec::codecForName("UTF-8"));
	QTextCodec::setCodecForCStrings(QTextCodec::codecForName("UTF-8"));
*/
	QCoreApplication::setOrganizationName(COMPANY);
	QCoreApplication::setApplicationName(APPLICATION_NAME);
    QCoreApplication::setApplicationVersion(SDRANGEL_VERSION);

#if QT_VERSION >= 0x050600
	QApplication::setAttribute(Qt::AA_EnableHighDpiScaling); // DPI support
    QCoreApplication::setAttribute(Qt::AA_UseHighDpiPixmaps); //HiDPI pixmaps
#endif
```
`QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);` at least should be moved before `QApplication a(argc, argv);`

**Comment by f4exb** [comment]
Issue #323 could be related

**Comment by f4exb** [comment]
Implemented in v5.6.1 and v4.14.7
